src/js/utils/is-in-viewport.js

```diff
@@ -1,11 +1,14 @@
 // https://stackoverflow.com/a/7557433
-module.exports = function isElementInViewport (el) {
+function isElementInViewport (el, win=window,
+                              htmlEl=document.documentElement) {
```

Comment:
Would it make sense to have `win` and `htmlEl` be optional, defaulting to `window` and `documentElement` if they don't exist?